src/methods/checkLang.ts

```diff
@@ -6,23 +6,25 @@ export declare namespace checkLang {
 
 export const checkLang = (): checkLang.langCode => {
   const mylang = document.getElementById("mylang")
+  if (!mylang) return "ja"
 
-  if (mylang) {
-    if (mylang.className.includes("mylang-ja")) {
+  return checkLangByDOMElement(mylang) ?? "ja"
+}
+
+const checkLangByDOMElement = (
+  mylang: HTMLElement
+): checkLang.langCode | undefined => {
+  switch (mylang.className) {
+    case "mylang-ja":
       return "ja"
-    } else if (mylang.className.includes("mylang-en")) {
+    case "mylang-en":
       return "en"
-    } else if (mylang.innerText.includes("English")) {
-      // mylangのクラス名がサービス側で変更されたケースがあったため、フォールバック
+  }
+
+  switch (mylang.innerText) {
+    case "日本語":
       return "ja"
-    } else if (mylang.innerText.includes("日本語")) {
-      // mylangのクラス名がサービス側で変更されたケースがあったため、フォールバック
+    case "English":
       return "en"
-    } else {
-      // 日本語をデフォルトに
-      return "ja"
-    }
   }
```

Comment:
Logic around here should be unchanged, because this change lowers the _flexibility_ to the updates (by the service) to the elements structure: imagine if the inner text was changed to `[日本語]` someday. These lines are intended to be a _fallback_ when the solid detection with `className` above failed (of course it means some changes to the element structure), so we should provide logic with maximum redundancy here.